pkg/obs/services/computes.go

```diff
@@ -0,0 +1,167 @@
+package services
+
+import (
+	"fmt"
+	"github.com/huaweicloud/huaweicloud-csi-driver/pkg/config"
+	"github.com/huaweicloud/huaweicloud-sdk-go-obs/obs"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func GetBucket(c *config.CloudCredentials, bucket string) (string, error) {
+	exists, err := CheckBucketExists(c, bucket)
+	if err != nil || !exists {
```

Comment:
if err != nil {
 return "", err
}
if !exists {
 return "", nil
}
this may be clearer

Reply:
done